style.css

```diff
@@ -0,0 +1,771 @@
+  flex-direction: column;
+  padding: 30px;
+  list-style: none;
+  margin-top: 20px;
+}
+
+.nav-menu {
+  background-color: #ec5242;
+  opacity: 0.9;
+  height: 100vh;
+  margin-top: -72px;
+  position: absolute;
+  width: 100%;
+  display: none;
+}
+
+.nav-menu.active {
+  display: block;
+}
+
+.nav-menu ul li {
+  margin: 20px;
+  font-size: 35px;
+}
+
+.nav-menu ul li a {
+  text-decoration: none;
+  color: #272a31;
+  font-weight: bold;
+}
+
+.fa-times {
+  color: #272a31;
+  float: right;
+  margin-right: 40px;
+  margin-top: 10px;
+}
+
+.product-list {
+  margin-bottom: 15px;
+}
+
+/* Desketop version */
+
+/* first page */
+
+@media screen and (min-width: 768px) {
+  .nav-icons {
+    display: flex;
+    justify-content: end;
+    padding: 12px;
+    color: #fff;
+    background-color: #272a31;
+  }
+
+  .nav-icons ul {
+    display: flex;
+    gap: 40px;
+    margin-right: 330px;
+  }
+
+  .nav-icons ul li {
+    list-style: none;
+  }
+
+  .nav-icons a {
+    text-decoration: none;
+    color: #fff;
+    font-size: 12px;
+  }
+
+  .fa-bars {
+    display: none;
+    visibility: hidden;
+    margin: 0;
+  }
+
+  .nav-bar {
+    display: flex;
+    justify-content: space-around;
+    background-color: #fff;
+    padding: 10px;
+  }
+
+  .nav-links {
+    display: flex;
+    gap: 45px;
+    font-weight: bold;
+    align-items: center;
+  }
+
+  .nav-links a {
+    text-decoration: none;
+    font-size: 16px;
+  }
+
+  .click {
+    width: 50px;
+  }
+
+  .flawless {
+    color: #ec5242;
+    border: 4px solid #ec5242;
+    padding: 10px;
+  }
+
+  .flawless:hover {
+    border: 3px solid #272a31;
+  }
+
+  .LINK {
+    color: #272a31;
+  }
+
+  .LINK:hover {
+    color: #ec5242;
+  }
+
+  /* header */
+
+  .main {
+    flex-direction: column;
+    margin-left: 250px;
+    width: 45%;
+    gap: 35px;
+  }
+
+  h1 {
+    font-size: 60px;
+  }
+
+  q {
+    font-size: 40px;
+  }
+
+  .description {
+    font-size: 18px;
+    padding: 18px;
+    line-height: 30px;
+  }
+
+  .promotion {
+    font-size: 34px;
+  }
+
+  /* Products */
+
+  .skincare {
+    flex-direction: column;
+    width: 16%;
+    margin: 3px;
+    text-align: center;
+  }
+
+  .skincare:hover {
+    border: 2px solid #fff;
+  }
+
+  .product-list {
+    display: flex;
+    justify-content: center;
+    margin: 50px;
+  }
+
+  .title-underline {
+    width: 5%;
+  }
+
+  .explication {
+    font-size: 14px;
+    max-width: 80%;
+    text-align: center;
+  }
+
+  .product-image {
+    height: 65px;
+    width: 70px;
+  }
+
+  .button {
+    padding: 10px;
+    background-color: transparent;
+    color: #fff;
+    font-size: 25px;
+    text-decoration: underline;
+    line-height: 150px;
+  }
+
+  /* Artists */
+
+  .artist-description {
+    display: flex;
+    flex-direction: row;
+    margin-top: 45px;
+    width: 100%;
+  }
+
+  .artist-wrapper {
+    width: 70%;
+    display: flex;
+    justify-content: center;
+    flex-wrap: wrap;
+    margin: 0 auto;
+  }
```

Comment:
- [ ] Please consider increasing the width of the container to at least 90% to ensure that the cards and the contents match the Figma design ☺️.

Your design at around 860px😎 
![image](https://github.com/mariegrace31/capstone-project/assets/99394374/71b11714-4aa4-4f98-ba2b-2e01bf40e899)

Original design 😎 
![image](https://github.com/mariegrace31/capstone-project/assets/99394374/9bf5e8f2-350c-47e5-b090-43a096d940d8)